src/reader/crontab_reader.rs

```diff
@@ -1,12 +1,120 @@
 extern crate regex;
-use crate::{RawCron, Reader};
+use crate::{RawCron, TaCron, Reader, TimeFieldValue};
+//use TaCron;
 use regex::Regex;
 use std::fs;
+use regex::Captures;
 
 pub struct CrontabReader {
     file: String,
 }
 
+
+// If regex match, return the result of function "f"
+struct FieldHandler {
+    regex: Regex,
+    f: fn(Captures) -> TimeFieldValue,
+}
+
+fn parse_field(field: &String, field_handlers: &[&FieldHandler]) -> Vec<TimeFieldValue> {
```

Comment:
`fn parse_field` and `field_handler` are generic imho. A cron in a DB row will have the same format so it will use the same parser, wdyt ?

Reply:
I was not sure if it's generic or not depending on the reader module...
Anyway I had troubles letting them in `mod.rs` and calling them in `crontab_reader.rs`, so I moved them there at least temporarly... You can give a shot to move them back if you want :)

Reply:
could you have a look please ? 4bc1f3b

Reply:
Awesome bro :sunglasses:

Reply:
I let you approve+merge